bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/OpenSprinklerBindingConstants.java

```diff
@@ -32,12 +34,13 @@
     public static final String CMD_ENABLE_MANUAL_MODE = "mm=1";
     public static final String CMD_DISABLE_MANUAL_MODE = "mm=0";
     public static final String CMD_PASSWORD = "pw=";
-    public static final String CMD_STATION = "sid=";
+    public static final String CMD_STATION = "&sid=";
     public static final String CMD_STATION_ENABLE = "en=1";
```

Comment:
It's very inconsistent to have the parameter separator (&) for the CMD_STATION_DISABLE, but not fot the ENABLE one. I would probably also prefer to add the separator when building the URL, instead of here in a constant (as it might, in some cases, also be a question mark (?))

Reply:
DONE: Agree and have made the change.